packages/gestalt/src/Box.js

```diff
@@ -493,7 +493,7 @@ const propToFn = {
         // default: none
       })(value),
       fromClassName(borders.solid),
-      fromClassName(colors.lightGrayBorder),
+      fromClassName(borders.borderColorLightGray),
```

Comment:
Looks like we were defining the same CSS class twice, once in `Colors.css` and the other time in `Borders.css`